modules/microshift-greenboot-pre-rollback-log.adoc

```diff
@@ -0,0 +1,50 @@
+
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-access-pre-rollback-check_{context}"]
+= Accessing pre-rollback health check output in the system log
+
+You can access the output of health check scripts in the system log. For example, to check the results of a pre-rollback script, use the following procedure.
+
+.Procedure
+
+* To access the results of a pre-rollback script, run the following command:
```

Comment:
This information should go in the RHEL docs https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/managing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images#rolling_back_rhel_for_edge_images

Reply:
Same comment here as above.